JoinColony/block-ingestor
# [Verified members] feat: block ingestor can now catch MetadataDelta events and parse operations, implement add verified members

## Description
Added a new handler for the `ColonyMetadataDelta` event. 
The handler tries to parse the event via type guards, it currently only has support for adding verified members.
Using this approach it should be easy to implement removing of verified members.

## How to test
Easiest way is to:
1. Stop the block ingestor while running `npm run dev`, and run this branch instead.
2. spin up the CDapp on [this branch](https://github.com/JoinColony/colonyCDapp/pull/1773).
3. Choose a member that hasn't been verified yet and copy their address 
4. go to the colony dashboard home, where the temporary component for verifying members lives, paste the address and click `Verify`
![image](https://github.com/JoinColony/block-ingestor/assets/23449297/57641b70-eaa8-416c-944c-06d58df3f982)
5. Check logs to see that block-ingestor processed the transaction
6. Use GraphQL query `getVerifiedMembersByColony` to check which users are added. The user you have pasted in above should show up in the `items` array

Resolves https://github.com/JoinColony/block-ingestor/issues/160
